Week3/homework/step2-7.js

```diff
@@ -21,3 +21,5 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
+// When I call the Function directly I can pass the value as a parameter and if I want to change the value of the argument which is inside the function then it will not affect the variable which is exist outside the scope
+// By reference way the inside value of the scope will affect the object or the array which is outside the scop
```

Comment:
I think you sort of grasped the idea. It is the difference between 'passing by value' and 'passing by reference'.